core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts

```diff
@@ -328,15 +328,23 @@ angular.module('oppia').directive('topicEditorTab', [
           };
 
           $scope.updatePracticeTabIsDisplayed = function(
-              newPracticeTabIsDisplayed) {
-            if (
-              newPracticeTabIsDisplayed !==
-              $scope.topic.getPracticeTabIsDisplayed()) {
-              TopicUpdateService.setPracticeTabIsDisplayed(
-                $scope.topic, newPracticeTabIsDisplayed);
```

Comment:
Let's add these tabs spacing back. Were these purposely removed?

Reply:
no it was just a mistake from us...